examples/cassandra/cassandra-controller.yaml

```diff
@@ -26,7 +26,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v6
+          image: gcr.io/google_containers/cassandra:v7
```

Comment:
Please see the TODO in the Makefile :P

(and its build using `mvn package` in the `.../java/` directory)